decidim-core/app/models/decidim/authorization.rb

```diff
@@ -49,8 +49,7 @@ def granted?
 
     # Returns true if the authorization is renewable by the participant
     def renewable?
-      return unless workflow_manifest
-      return unless renewable_at < Time.current
+      return unless workflow_manifest && renewable_at < Time.current
```

Comment:
Sorry, I didn't explain myself well. 🙈 

I understand that a return statement at the beginning of a method could be called "a guard" when it performs some kind of validation before executing the "real" method logic. I don't know if there is a better or clearer definition for it.

So, what I was trying to say is that checking if there is a `workflow_manifest` could be implemented with a guard while the response to the answer `renewable?` will depend on the manifest `renewable` setting and the last renewal date. Something like:

```ruby
def renewable?
  # the guard
  return unless workflow_manifest

  # the "real" method logic
  workflow_manifest.renewable && renewable_at < Time.current
end
```

Sorry for being so nit-picky. 😬 
Do you agree with this approach?

Reply:
🤦‍♂️ my fault, I understood the other way round 😂